.python-version

```diff
@@ -1 +0,0 @@
-3.8.12/envs/venv
```

Comment:
I assume this file was accidentally committed, since it theoretically conflicts with everybody else's local `.python-version`, and none of our other repos have this file committed.